values/knative-serving/knative-serving-cr.gotmpl

```diff
@@ -53,12 +53,12 @@ spec:
     domain:
       {{ $v.cluster.domainSuffix }}: ''
     istio:
-      gateway.knative-serving.knative-ingress-gateway: istio-ingressgateway-public.istio-system.svc.cluster.local
+      gateway.knative-serving.knative-ingress-gateway: {{ $v._derived.ingressGatewayPrefix }}-public.istio-system.svc.cluster.local
```

Comment:
I saw few time composing the gateway name "{{ $v._derived.ingressGatewayPrefix }}-public."

Would it be possible to set it in derived.gotmpl file and propagate this value here and to team-ns chart?

Reply:
Done.